configure

```diff
@@ -6481,8 +6469,14 @@ else # shared is disabled
   PY_ENABLE_SHARED=0
   case $ac_sys_system in
     CYGWIN*)
-          BLDLIBRARY='$(LIBRARY)'
-          LDLIBRARY='libpython$(LDVERSION).dll.a'
+      BLDLIBRARY='$(LIBRARY)'
+      LDLIBRARY='libpython$(LDVERSION).dll.a'
+          ;;
+    Linux*|GNU*|NetBSD*|FreeBSD*|DragonFly*|OpenBSD*|VxWorks*)
+	  BLDLIBRARY='-Wl,--whole-archive $(LDLIBRARY) -Wl,--no-whole-archive'
+          ;;
+    Darwin*)
+	  BLDLIBRARY='-Wl,-all_load $(LDLIBRARY)'
```

Comment:
You are mixing tabs and spaces. Could you please use spaces-only and ``AS_CASE``?

```suggestion
  AS_CASE([$ac_sys_system],
    [CYGWIN*], [
      BLDLIBRARY='$(LIBRARY)'
      LDLIBRARY='libpython$(LDVERSION).dll.a'
    ], 
    [Linux*|GNU*|NetBSD*|FreeBSD*|DragonFly*|OpenBSD*|VxWorks*], [
      BLDLIBRARY='-Wl,--whole-archive $(LDLIBRARY) -Wl,--no-whole-archive'
    ],
    [Darwin*], [BLDLIBRARY='-Wl,-all_load $(LDLIBRARY)']
  )
```

Reply:
> You are mixing tabs and spaces.
    I never use tabs in my local in fact. I am confused too. this patch's configure file in my local.
    Let me check it again.
![image](https://user-images.githubusercontent.com/9103416/144740467-fd9159a6-1e0d-438f-b0db-9e18724b66b1.png)

Reply:
OK, Let me check it again.

Reply:
OK, I forget my automative tabs when I press the `enter`.
Updated, thanks.